buildSrc/build.gradle.kts

```diff
@@ -8,8 +8,10 @@ dependencies {
     implementation("de.westnordost:countryboundaries:1.5")
     implementation("com.esotericsoftware.yamlbeans:yamlbeans:1.15")
     implementation("org.jsoup:jsoup:1.14.2")
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.2")
```

Comment:
Hm now we have two JSON-parsing libraries here. Klaxon and Kotlinx-serialization. Oh well, I guess dependency frugality doesn't matter much for build scripts.